libnavui-dropin/build.gradle

```diff
@@ -33,6 +33,11 @@ android {
     testOptions {
         unitTests.returnDefaultValues = true
         unitTests.includeAndroidResources = true
+        unitTests.all {
+            jacoco {
+                excludes = ['jdk.internal.*']
+            }
+        }
```

Comment:
Rebasing on top of `main` probably should have been enough, because we already have that setup in jacoco.gradle.

Reply:
yep, rebased and removed it from the gradle file.